packages/http-server/src/server.ts

```diff
@@ -71,7 +71,7 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
     request.log.info({ input }, 'Request received');
     try {
       const operationSpecificConfig = getHttpConfigFromRequest(input);
-      const mockConfig = { ...opts.config.mock, ...operationSpecificConfig };
+      const mockConfig = opts.config.mock === false ? false : { ...opts.config.mock, ...operationSpecificConfig };
```

Comment:
do we have a way to `not to mock` from the CLI? Seems to me like it's only in the tests that we can force the forwarder to be used. Opts passed to `createHttpServer` stays the same in this PR (are not changed). Should we enable the forwarder from CLI or is this PR only a part of changes for the proxying?

Reply:
That's correct. This PR right now will only enable this from the Client JS API; once we'll figure out how the proxy should work we'll then enable the relative command.